certified-connectors/Tikit/apiDefinition.swagger.json

```diff
@@ -1741,1435 +1779,2903 @@
-        },
-        "GroupId": {
-          "description": "Group Id",
-          "type": "integer",
-          "format": "int32",
-          "x-ms-summary": "Group Id"
-        },
-        "EscalationId": {
-          "description": "Escalation Id",
-          "type": "integer",
-          "format": "int32",
-          "x-ms-summary": "Escalation Id"
+        "summary": "Update Approval",
+        "description": "Update Approval",
+        "operationId": "UpdateApproval",
+        "parameters": [
+          {
+            "name": "x-requested-by",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "PowerAutomate",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Approval Id",
+            "x-ms-summary": "Approval Id"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Title": {
+                  "type": "string",
+                  "description": "Title",
+                  "title": "Title"
+                },
+                "AdditionalDetails": {
+                  "type": "string",
+                  "description": "Additional Details",
+                  "title": "Additional Details"
+                },
+                "RequiredByAll": {
+                  "type": "string",
+                  "description": "RequiredByAll",
+                  "title": "Required By All",
+                  "enum": [
+                    "True",
+                    "False"
+                  ]
+                },
+                "Approvers": {
+                  "type": "string",
+                  "description": "Approver(s) email address separated by comma",
+                  "title": "Approvers"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/GetPowerAutomateTaskStatus": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Task Status Schema",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "value": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/definitions/ServiceDesk.Core.Models.PowerAutomateTask"
+                  }
+                }
+              }
+            }
+          }
         },
-        "CategoryId": {
-          "description": "Category Id",
-          "type": "integer",
-          "format": "int32",
-          "x-ms-summary": "Category Id"
+        "produces": [
+          "application/json"
+        ],
+        "summary": "Get Power Automate Task Status",
+        "operationId": "GetPowerAutomateTaskStatus",

```

Comment:
The keyword Power Automate is not allowed, can you please remove
The value 'summary' contains at least one of the restricted words: 'power apps, power automate, logic apps'. ValuePath: /paths/~1Ticket({Id})~1GetPowerAutomateTasks/get/summary